src/BSeries.jl

```diff
@@ -1447,4 +1448,437 @@ include("latexify.jl")
     include("precompile.jl")
 end
 
+
```

Comment:
Please clean up this PR. It looks to me like there is a huge amount of changes that is not really related to what you would like to do in this PR - in particular all changes to this file as well as `src/Energy_Preserving.jl`?